src/App.js

```diff
@@ -22,34 +19,31 @@ function App() {
 
   // if (error) console.log(error);
   // for Get user------------------------------------------>
-
-  useEffect(()=>{
-    getData();
-  }, [])
   const [data, setData] = useState([]);
-  const [loading, setLoading]=useState(false);
-
-   const getData = async ()=>{
-
+  const [loading, setLoading] = useState(false);
 
+   const getData = async () => {
     try{
      const res = await axios.get("https://randomuser.me/api?results=1")
      setData(res.data.results)
      setLoading(true)
```

Comment:
Wouldn't you set loading to `true` while you're doing the fetch? This looks to me like we should be finished loading at this point.